RNTVUIKit/RCTTVPosterView.h

```diff
@@ -14,6 +14,7 @@ API_AVAILABLE(tvos(12.0))
 @interface RCTTVPosterView : TVPosterView
 
 @property(nonatomic, nullable, copy) NSString *imageURL;
+@property(nonatomic, nullable, copy) NSString *imageURI;
```

Comment:
I'd like to go even further here, and have a `source` property just like the `Image` component.  This would make it more consistent, and more flexible to deal with different image sources.  In react-native repo, the relevant files are

- `Libraries/Image/Image.ios.js`
- `Libraries/Image/ImageSourcePropType.js`
- `React/Base/RCTImageSource.[hm]`

What do you think?  Would that be overkill?

Reply:
I took at look at it this weekend but need some more time to investigate how this would work.  One thing that is confusing me overall about the PosterView is styling.  It seems like the component allows me to pass in a style property which only applies styling to the image itself.  But the component also accepts `title` and `subtitle` properties.  How would one style these?  I dug through `RCTTVPosterView` as well as the code in `react-native-tvuikit` but I couldn't find relevant code that documents this.